dev/io.openliberty.cdi.4.0.internal_fat/fat/src/io/openliberty/cdi40/internal/fat/config/beansxml/AllBeansServlet.java

```diff
@@ -29,14 +27,11 @@ public class AllBeansServlet extends FATServlet {
     private RequestScopedBean requestScopedBean;
 
     @Inject
-    private Instance<UnannotatedBean> unannotatedBean;
+    private UnannotatedBean unannotatedBean;
 
     @Test
     public void testAllBeansInjected() {
         assertNotNull(requestScopedBean);
         assertEquals("RequestScopedBean", requestScopedBean.test());
-
-        assertTrue(unannotatedBean.isResolvable());
-        assertEquals("UnannotatedBean", unannotatedBean.get().test());
     }
```

Comment:
Should you be calling the `unannotatedBean` here? I don't expect it to ever fail if the app successfully starts, but it would be consistent with the test for `requestScopedBean`.

Reply:
done